x-pack/plugins/translations/translations/zh-CN.json

```diff
@@ -161,7 +161,6 @@
     "common.ui.aggTypes.ipRanges.toLabel": "到",
     "common.ui.aggTypes.ipRanges.useCidrMasksButtonLabel": "使用 CIDR 掩码",
     "common.ui.aggTypes.ipRanges.useFromToButtonLabel": "使用“从”/“到”",
-    "common.ui.aggTypes.jsonInputLabel": "JSON 输入",
```

Comment:
I removed this label because sentence case was applied: `JSON Input` -> `JSON input`